nat-lab/data/core_api/README.md

```diff
@@ -0,0 +1,4 @@
+# Self-Signed Certificate (SSC)
+
+This folder contains **self-signed certificate**. \
+Please note that this certificate is intended **only for testing purposes** in nat-lab environment.
```

Comment:
nit: It would be nice to add here the exact command(s) used to generate the certificates.